src/app/data-planes/sources.ts

```diff
@@ -74,6 +77,7 @@ export const sources = (api: KumaApi, can: Can) => {
           'outbound_passthrough',
           'access_log_sink',
           'ads_cluster',
+          'meshtrace_zipkin',
```

Comment:
We have https://github.com/kumahq/kuma/pull/8654 so looks like you can already add a blanket (starts with `_`)

Reply:
Awesome 🎉